frontend/src/queries/nodes/InsightQuery/utils/queryNodeToFilter.ts

```diff
@@ -282,6 +282,7 @@ export const queryNodeToFilter = (query: InsightQueryNode): Partial<FilterType>
         delete queryCopy.pathsFilter?.maxEdgeWeight
         delete queryCopy.pathsFilter?.funnelPaths
         delete queryCopy.pathsFilter?.funnelFilter
+        delete queryCopy.pathsFilter?.funnelActorsQuery
```

Comment:
Note: When removing the `funnelFilter` key from the query node, we'll need to build the filter from the `funnelActorsQuery` key instead.

Reply:
Backend side we also have `posthog/hogql_queries/legacy_compatibility/filter_to_query.py` that should be updated accordingly, since we probably will use that function to migrate existing insights.